CMSgov/mint-app
# [EASI-2676] Merge cmmi groups sig and pphg

# EASI-2676

## Changes and Description

### 
- Change 1
- Change 2

### Front End Changes
- Updated Basics i18n translate files
- Updated `translateCmsCenter` and `translateCmmiGroups` to use translation 
- Updated tests and snaps 

## How to test this change

1. Go to Model basics in any model plan
2. Ensure that the `State and Population Health Group (SPHG)` and `To be determined` are now options
3. Select those and ensure the data is saving these new values



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
